File: src/main/java/com/example/dashboard/model/WebService.java
package com.example.dashboard.model;

import javax.persistence.*;

@Entity
@Table(name="website")
public class WebService {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name="url")
    private String url;

    @Column(name="status")
    private String status;

    public void setId(Long id) {
        this.id = id;
    }

    public void setUrl(String url) {
        this.url = url;
    }
    public void setStatus(String status) {
        this.status = status;
    }
    public Long getId() {
        return id;
    }
    public String getUrl() {
        return url;
    }
    public String getStatus() {return status;}


}

File: src/main/java/front/frontendDashboard/test/src/app/webservice.ts
export class Webservice {
  private _id : number;
  private _url : string;
  private _status : string

  constructor(id:number,url:string,status : string) {
    this._id=id;
    this._url=url;
    this._status=status;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get url(): string {
    return this._url;
  }

  set url(value: string) {
    this._url = value;
  }

  get status(): string {
    return this._status;
  }

  set status(value: string) {
    this._status = value;
  }
}
